Makefile

```diff
@@ -48,16 +46,12 @@ DEV_TAGS := $(if ${tags},$(DEV_TAGS) ${tags},$(DEV_TAGS))
 # We only return the part inside the double quote here to avoid escape issues
 # when calling the external release script. The second parameter can be used to
 # add additional ldflags if needed (currently only used for the release).
-make_ldflags = $(2) -X $(PKG)/build.Commit=$(COMMIT) \
-	-X $(PKG)/build.CommitHash=$(COMMIT_HASH) \
-	-X $(PKG)/build.GoVersion=$(GOVERSION) \
-	-X $(PKG)/build.RawTags=$(shell echo $(1) | sed -e 's/ /,/g')
+make_ldflags = $(2) -X $(PKG)/build.Commit=$(COMMIT) 
 
 DEV_GCFLAGS := -gcflags "all=-N -l"
 LDFLAGS := -ldflags "$(call make_ldflags, ${tags}, -s -w)"
 DEV_LDFLAGS := -ldflags "$(call make_ldflags, $(DEV_TAGS))"
```

Comment:
Can remove the additional parameter now and use `$(1)` in `make_ldflags`. That also allows us to unify `DEV_LDFLAGS` and `ITEST_LDFLAGS` into the same variable (maybe also rename `LDFLAGS` to `RELEASE_LDFLAGS` and the others into a combined `DEV_LDFLAGS`)?

Reply:
Not sure about this one.